### PR TITLE
修复观察网、CCTV、CNBeta、新华网作者、时间取值，新增国家广播电视总局及参考消息源

### PR DESCRIPTION
**Pull request by yuxinliu-alex**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```
/guancha/review
/cctv/china
/cnbeta
/cankaoxiaoxi/news/military_news
/news/whxw
/nrta/news/important
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/thepaper/utils.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const cheerio = require('cheerio');
+const date = require('@/utils/date');
```
Comment (TonyRL) [inline lib/routes/thepaper/utils.js:2]: Please use `parseDate` from `@/utils/parse-date`

File: lib/routes/thepaper/utils.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+const got = require('@/utils/got');
+
+module.exports = {
+    ProcessFeed: async (query, link, ctx) => {
+        const res = await got.get(link);
```
Comment (TonyRL) [inline lib/routes/thepaper/utils.js:7]:
```suggestion
        const res = await got(link);
```

File: lib/routes/thepaper/utils.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+        const $ = cheerio.load(res.data);
+        const list = $(query).slice(0, 10).get();
+
+        return Promise.all(
+            list.map(async (item) => {
+                const $ = cheerio.load(item);
+                const itemUrl = `https://m.thepaper.cn/${$(item).find('a').eq(0).attr('href')}`;
+                const cache = await ctx.cache.get(itemUrl);
```
Comment (TonyRL) [inline lib/routes/thepaper/utils.js:16]: Please use `ctx.cache.tryGet(k, v)` instead.

File: lib/routes/thepaper/utils.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+                }
+
+                const single = {
+                    title: content('title').text(),
+                    link: itemUrl,
+                    description,
+                    pubDate,
+                    author: auth,
+                };
+                return Promise.resolve(single);
```
Comment (TonyRL) [inline lib/routes/thepaper/utils.js:48]:
```suggestion
                return single;
```

File: lib/v2/cankaoxiaoxi/index.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
+const got = require('@/utils/got');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const cheerio = require('cheerio');
+const utils = require('@/v2/icac/utils');
```
Comment (TonyRL) [inline lib/v2/cankaoxiaoxi/index.js:5]: Do not use utilities for other routes

File: lib/v2/cankaoxiaoxi/index.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    ctx.state.data = {
+        title: `${nodes[category].title} 参考消息`,
+        link: currentUrl,
+        description: '参考消息',
+        language: ctx.params.lang ? utils.LANG_TYPE[ctx.params.lang] : utils.LANG_TYPE.sc,
+        item: items,
+    };
+};
+
+// module.exports = async (ctx) => {
```
Comment (TonyRL) [inline lib/v2/cankaoxiaoxi/index.js:90]: Please remove commented code of another route

File: lib/v2/nrta/news.js [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    update: 114,
+};
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category || '';
+
+    const rootUrl = 'http://www.nrta.gov.cn';
+
+    let currentUrl = '';
+    currentUrl = `${rootUrl}/col/col${categories[category]}/index.html`;
```
Comment (TonyRL) [inline lib/v2/nrta/news.js:19]: Use `const` if `currentUrl` will not be reassigned

File: lib/v2/nrta/router.js [hunk 8]
```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
```
Comment (TonyRL) [inline lib/v2/nrta/router.js:1]: This route should be added into `lib/v2/gov` as `.gov`-related website are added there.

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please do not squash route fixes and new routes into one PR.

**Comment by yuxinliu-alex** [comment]
> Please do not squash route fixes and new routes into one PR.

seperate the fixes into #9561 #9562 #9559 #9563,

seperate the new routes into #9566 #9564 .

Best

**Pull request closed without merging**